app/models/generic_work.rb

```diff
@@ -2,6 +2,8 @@
 
 class GenericWork < ActiveFedora::Base
   include ::Hyrax::WorkBehavior
+  include IiifPrint::SetChildFlag
+  include IiifPrint.model_configuration(pdf_split_child_model: Image)
```

Comment:
Should line 6 go on the Image model?

Reply:
or, is this saying that the child work types for a generic work parent will be images? @laritakr